gammapy/background/background_estimate.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from .ring import ring_area_factor
+from ..region import find_reflected_regions
+
+__all__ = [
+    'BackgroundEstimate',
+    'ring_background_estimate',
+    'reflected_regions_background_estimate',
+]
+
+
+class BackgroundEstimate(object):
+    """Cointainer class for background estimate
+
+    This is meant to hold the result from a region based background estimation
+    for one observation.
+
+    Parameters:
+    -----------
+    off_events : `~gammapy.data.EventList`
+        Background events
+    off_region : `~astropy.regions.SkyRegion`
+        Background estimation region
+    alpha : float
+        Background scaling factor
+    tag : str
+        Background estimation method
+    """
+    def __init__(self, off_region, off_events, alpha, tag='default'):
+        self.tag = tag
```

Comment:
Move this line to the end of the function. (Code is easier to read and maintain if things are always in the same order, i.e. as in the docstring and function signature)